### DISCUSSION TITLE
How to include additional content in the sidebars?

### DISCUSSION THREAD

**Discussion by NicolasMassart** [squidfunk/mkdocs-material#2130] [Q&A] [answered]
RTD Ethical Ads are https://ethical-ad-client.readthedocs.io/
I tried to include the following at then end of the nav.html partial:

```html
<div id="readthedocs-ethical-ads" class="flat" data-ea-publisher="readthedocs" data-ea-type="image" data-ea-campaign-types="community|house"></div>
```

But I have issues with the scroll and the page stops being scrollable at some point, I think it's linked to the left nav size calculation that's not meant to include an injected ad banner.

<img width="1200" alt="Screenshot 2020-12-15 at 14 30 34" src="https://user-images.githubusercontent.com/4677568/102221385-656b6480-3ee2-11eb-8704-540f2ec3e519.png">

Does anyone already succeeded in this? It would save me a lot of CSS/HTML headaches... (not my core skill clearly)

Thanks.

**Comment by squidfunk** [comment, marked as answer]
Just place it in `.md-sidebar__scrollwrap`, and the size should be computed correctly.

**Comment by NicolasMassart** [comment]
I have an issue of the scroll being impossible on the content part when the nav is getting bigger than the page.
You can see this here https://hyperledger-besu--576.org.readthedocs.build/en/576/Reference/CLI/CLI-Subcommands/
If you scroll a bit in the left nav, then at some point the content becomes unable to scroll... I don't really explain this. Maybe it's not linked to you theme but to [the changes I made on it](https://github.com/NicolasMassart/besu-docs/tree/575-get-ethical-ads-back/custom_theme) but if you had a piece of advise that would help. Thanks.

**Reply by squidfunk** [reply]
This is a bug in Chrome that I already reported and hoped would be fixed at some point. However, that was half a year ago, so I don't really know when this will be fixed. It seems that the Chrome team has other priorities. It's rather indeterministic and related to scroll snapping. You can reset all `scroll-snap-*` properties on the `*__scrollwrap` elements and it should be gone.

**Reply by squidfunk** [reply]
I changed the title to be more generic so users can find it easier.

**Reply by squidfunk** [reply]
6.2.2 removes the scroll snapping for the desktop view (= sidebars), as the implementation bugs in Chrome are really annoying. I hope they fix it soon.
